tmp/repositories.json

```diff
@@ -0,0 +1,38 @@
+[
```

Comment:
Your `.gitignore` diff suggests this file should not be added.

Reply:
I have removed the `tmp` file and squashed the commits.